packages/ecmascript/src/Module.ts

```diff
@@ -40,15 +108,68 @@ export interface ModuleExportConstructor {
 }
 
 export interface ModuleExport {
+	/**
+	 * Prepares an element to be exported with the given name upon completion of the export builder.
+	 * @param {string} name - name of the exported element 
+	 * @param {any} element 
+	 * @returns {ModuleExport} the builder instance
+	 */
 	named(name: string, element: any): ModuleExport;
+	/**
+	 * Prepares an element to be exported as default upon completion of the export builder.
+	 * @param {any} element 
+	 * @returns {ModuleExport} the builder instance
+	 */
 	default(element: any): ModuleExport;
+	/**
+	 * Completes the export builder.
+	 * @returns {ModuleElementList} object containing the prepared elements for export.
+	 */
 	build(): ModuleElementList;
 }
 
 export interface ModuleElementList {
 	[name: string]: any;
 }
 
+/**
+ * Key of the Module attribute that holds the last set error handler via {@link Module.setModuleErrorHandler}.
+ */
+const MODULE_ERROR_HANDLER_KEY = "__onError";
+
+/**
+ * Predefined handlers for errors on loading/importing a module or action - see {@link DefaultModuleErrorHandlers}
+ */
+const ModuleErrorHandlers = {
```

Comment:
Honestly I don't think this will ever change.. we give them a default and they can overwrite it if they want, imho

Reply:
Agreed. Keeping only the System.error option as default (will also address https://github.com/vmware/build-tools-for-vmware-aria/pull/345#discussion_r1713688659 )